client-internal/src/main/java/org/terracotta/angela/client/Tsa.java

```diff
@@ -350,7 +350,7 @@ public Tsa activateAll() {
         Agent.controller.configure(instanceId, terracottaServer, topology, null, tsaConfigurationContext.getClusterName(), null, cliEnv, false);
       });
 
-      final int maxWaitTimeMillis = 30000;
+      final int maxWaitTimeMillis = 10000;
```

Comment:
is it safe for jenkins node that are slower ? shouldn't this value be controllable or made larger instead ?

Reply:
The code before wasn't waiting at all (actual wait time never exceeded 100ms or so). 10s should be enough. Keeping it as is doesn't hurt us though.

I would prefer to not introduce another system property. This is way too specific for a generic angela-level system property.